### PR TITLE
Uhs branches and percentiles

### PR DESCRIPTION
**Pull request by joelridden**
### **Adding Branches and Percentiles to UHS charts**
This affected the project and non-project UI pages for UHS

**UHS updated to two pages**
One page shows original plot with all RP
The other page shows all branches and percentiles if retrieved for that project / non-project for a selected RP (Uses dropdown to select a different RP)
image

### CODE REVIEW

**Review by jasonmotha — APPROVED** [review]

**Review by jasonmotha — DISMISSED** [review]

**Review by jasonmotha — DISMISSED** [review]

File: frontend/src/components/Project/SeismicHazard/HazardViewerUHS.js [hunk 1]
```diff
@@ -145,38 +154,110 @@ const HazardViewerUHS = () => {
...
+                      value={
+                        localSelectedRP === undefined ||
+                        localSelectedRP === null
+                          ? []
+                          : { label: localSelectedRP, value: localSelectedRP }
+                      }
+                      onChange={(e) => setLocalSelectedRP(e.value)}
+                      options={createSelectArray(getSelectedRP().sort(function(a, b){return a-b}))}
+                      isDisabled={getSelectedRP().length === 0}
+                      menuPlacement="auto"
```
Comment (SnowNooDLe) [inline frontend/src/components/Project/SeismicHazard/HazardViewerUHS.js:232]: @joelridden We completely forgot about Projects

Comment (joelridden) [reply frontend/src/components/Project/SeismicHazard/HazardViewerUHS.js:232]: I'll do the same thing to projects then

File: frontend/src/components/Hazard/SeismicHazard/HazardViewerUHS.js [hunk 2]
```diff
@@ -174,39 +202,103 @@ const HazardViewerUHS = () => {
...
-          showErrorMessage.isError === false && (
-            <Fragment>
-              <UHSPlot
-                from={"non-projects"}
-                uhsData={uhsData}
-                nzs1170p5Data={uhsNZS1170p5Data}
-                showNZS1170p5={showUHSNZS1170p5}
-                extra={extraInfo}
+      <Tabs defaultActiveKey="allRP" className="pivot-tabs">
+        <Tab eventKey="allRP" title="Selected Return Periods">
```
Comment (SnowNooDLe) [inline frontend/src/components/Hazard/SeismicHazard/HazardViewerUHS.js:206]: Later on, let's have a brief chat with Tab's naming convention.
I don't think it's worth having a chat now and fix within this PR as this is something really trivial.

**Review by SnowNooDLe — DISMISSED** [review]

**Comment by claudio525** [comment]
This PR has a whole bunch of commits from Tom at the beginning, so I'm guessing some of this code is from Tom and hasn't been merged into `master_ea` separately? @SnowNooDLe @joelridden

**Comment by SnowNooDLe** [comment]
> This PR has a whole bunch of commits from Tom at the beginning, so I'm guessing some of this code is from Tom and hasn't been merged into `master_ea` separately? @SnowNooDLe @joelridden

@claudio525 Nah, I think the issue was @joelridden accidentally created a branch from `master_dev` and before he opened this PR, merged the `master_dev` instead of `master_ea`

Based on this [ReadME](https://github.com/ucgmsim/seistech_psha_frontend#readme), we should always create a branch from `master_ea` and if there were any changes to `master_ea`, we merge `master_ea` before opening PR.

**Comment by joelridden** [comment]
> > This PR has a whole bunch of commits from Tom at the beginning, so I'm guessing some of this code is from Tom and hasn't been merged into `master_ea` separately? @SnowNooDLe @joelridden
> 
> @claudio525 Nah, I think the issue was @joelridden accidentally created a branch from `master_dev` and before he opened this PR, merged the `master_dev` instead of `master_ea`
> 
> Based on this [ReadME](https://github.com/ucgmsim/seistech_psha_frontend#readme), we should always create a branch from `master_ea` and if there were any changes to `master_ea`, we merge `master_ea` before opening PR.

Correct I believe I did create the branch from master_dev accidentally

**Comment by jasonmotha** [comment]
Should we time-travel and apply these changes to master-ea then?

**Comment by SnowNooDLe** [comment]
> Should we time-travel and apply these changes to master-ea then?

I think by looking at the commit history, Joel did merge the `master_ea` which seems it is up-to-date?

**Comment by jasonmotha** [comment]
> > Should we time-travel and apply these changes to master-ea then?
> 
> I think by looking at the commit history, Joel did merge the `master_ea` which seems it is up-to-date?

It's just a question of do we want to incorporate those changes from master_dev into master_ea at this point in time.

**Review by SnowNooDLe — APPROVED** [review]

**Review by jasonmotha — APPROVED** [review]

**Pull request merged**